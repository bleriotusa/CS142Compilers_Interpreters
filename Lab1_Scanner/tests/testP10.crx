main(){
  int x = leTs + mAKe - uSE;
  short y = of - mIXEd + caps;
}

Supercalifragilisticexpialidocious
