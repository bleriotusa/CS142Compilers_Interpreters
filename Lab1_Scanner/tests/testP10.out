IDENTIFIER(main)(lineNum:1, charPos:1)
OPEN_PAREN(lineNum:1, charPos:5)
CLOSE_PAREN(lineNum:1, charPos:6)
OPEN_BRACE(lineNum:1, charPos:7)
IDENTIFIER(int)(lineNum:2, charPos:3)
IDENTIFIER(x)(lineNum:2, charPos:7)
ASSIGN(lineNum:2, charPos:9)
IDENTIFIER(leTs)(lineNum:2, charPos:11)
ADD(lineNum:2, charPos:16)
IDENTIFIER(mAKe)(lineNum:2, charPos:18)
SUB(lineNum:2, charPos:23)
IDENTIFIER(uSE)(lineNum:2, charPos:25)
SEMICOLON(lineNum:2, charPos:28)
IDENTIFIER(short)(lineNum:3, charPos:3)
IDENTIFIER(y)(lineNum:3, charPos:9)
ASSIGN(lineNum:3, charPos:11)
IDENTIFIER(of)(lineNum:3, charPos:13)
SUB(lineNum:3, charPos:16)
IDENTIFIER(mIXEd)(lineNum:3, charPos:18)
ADD(lineNum:3, charPos:24)
IDENTIFIER(caps)(lineNum:3, charPos:26)
SEMICOLON(lineNum:3, charPos:30)
CLOSE_BRACE(lineNum:4, charPos:1)
IDENTIFIER(Supercalifragilisticexpialidocious)(lineNum:6, charPos:1)
EOF(lineNum:7, charPos:1)
